src/dev-guide/express-tutorial.md

```diff
@@ -0,0 +1,105 @@
+# Express Tutorial
+
+Express can be used to handle requests to the api of our application, it can also be used as middleware which could help us to pass json to the front end of the application.
+
+
+
+Each express file requires the following head segment:
+
+```
+const express = require("express")
+```
+
+
+
+To make a new route, a route must be specified in the server.js file like:
+
+```
+const exampleRouter = require("./routes/example")
+app.use("/example", exampleRouter)
+```
+
+
+
+
+
+And create a matching new file in the routes folder with the name of your route (in this case example). 
+
+In this file you need to add the header segment above as well as declare the router:
+
+```
+const express = require ('express')
+const exampleRouter = express.Router()
+```
+
+
+
+At the end of the file you must export it:
+
+```
+module.exports = exampleRouter
+```
+
+
+
+In this file you can specify new methods for the api related to the pathway. This is done in the following format:app.get("/test1", (req, res) => {
+
+```
+    const test1 = {
+
+    "text" : "Hello World"
+
+    }
+
+    res.send(test1)
+
+})
+
+
+
+App.[request type](“path from route”, (req, res, next(optional)) => {
+
+    *method*
```

Comment:
I recommend using comments instead (`/* method */`) to indicate stuff the reader needs to fill in